sa/precertificates.go

```diff
@@ -162,7 +162,7 @@ func (ssa *SQLStorageAuthority) AddPrecertificate(ctx context.Context, req *sapb
 		// when the parent function ends. The rocsp client has a
 		// configurable timeout that can be set during creation.
 		rocspCtx := context.Background()
-		rocspTTL := ssa.clk.Now().Sub(parsed.NotAfter)
+		rocspTTL := parsed.NotAfter.Sub(ssa.clk.Now())
```

Comment:
I'm struggling to differentiate the result between these two calls. Why would the result of this call be preferable?

Reply:
We expect NotAfter to be in the future, so `now - notAfter` will be negative; `notAfter - now` will be positive.